File: OrderNet-Server/src/main/java/bitc/fullstack503/ordernetserver/dto/WHDTO.java
package bitc.fullstack503.ordernetserver.dto;

import lombok.Data;

@Data
//    물류센터 입고재고 DTO
public class WHDTO {

    private String partId; //.부품 아이디
    private String partName ; //부품 이름
    private String partCate; //부품 카테
    private int stockQuantity;// 부품 수량
    private int partPrice;// 부품 가격

    //-----------입고 DTO
    private int inboundQuantity; //입고 수량
    private String inboundDate; //입고 날짜

    //-----------------출구 DTO

    private int outboundId;
    private String warehouseId;
    private String orderId;
    private String outboundDate;

    private int totalQuantity;




}

File: OrderNet-APP/app/src/main/java/bitc/fullstack/app/dto/WHDTO.kt
package bitc.fullstack.app.dto

import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.Date

data class WHDTO(
    @SerializedName("orderId")
    var orderId: String = "",

    @SerializedName("orderDate")
    var orderDate: Date ,

    @SerializedName("orderDueDate")
    var orderDueDate: Date,

    @SerializedName("branchId")
    var branchId: String ="",

    @SerializedName("branchName")
    var branchName: String ="",

    @SerializedName("partId")
    var partId: String ="",

    @SerializedName("partCate")
    var partCate: String ="",

    @SerializedName("partName")
    var partName: String ="",

    @SerializedName("partPrice")
    var partPrice: Int =0,

    @SerializedName("stockQuantity")
    var stockQuantity: Int =0,

    @SerializedName("orderItemQuantity")
    var orderItemQuantity: Int =0,

    @SerializedName("totalQuantity")
    var totalQuantity: Int =0,

    @SerializedName("outboundDate")
    var outboundDate: Date,

    @SerializedName("warehouseId")
    var warehouseId: Int = 0
) : Serializable
